src/ColorSyncApp.js

```diff
@@ -7,6 +10,8 @@ class ColorSyncApp extends Component {
   render (){
     return (
       <div className="App">
+              <Header />
```

Comment:
You can pass the data from here to Header and then use it on the buttons component
In that way you dont call the data.js in every component